go.sum

```diff
@@ -480,6 +487,7 @@ k8s.io/api v0.17.2 h1:NF1UFXcKN7/OOv1uxdRz3qfra8AHsPav5M93hlV9+Dc=
 k8s.io/api v0.17.2/go.mod h1:BS9fjjLc4CMuqfSO8vgbHPKMt5+SF0ET6u/RVDihTo4=
 k8s.io/apiextensions-apiserver v0.17.2 h1:cP579D2hSZNuO/rZj9XFRzwJNYb41DbNANJb6Kolpss=
 k8s.io/apiextensions-apiserver v0.17.2/go.mod h1:4KdMpjkEjjDI2pPfBA15OscyNldHWdBCfsWMDWAmSTs=
+k8s.io/apimachinery v0.17.0/go.mod h1:b9qmWdKlLuU9EBh+06BtLcSf/Mu89rWL33naRxs1uZg=
```

Comment:
:grimacing: Is this coming in through `kind`? Do we need to worry about that potentially becoming out of sync with what we are pulling in from controller-runtime?

Reply:
Yes, although that's not a huge problem. In the past we have followed kind versions compatible with cluster API, for example when we added CRDv1, the kind version was also upgraded. 

In the future we might have to do a replace for apimachinery, or work with kind to match our version